backend/app/crud/crud_skill.py

```diff
@@ -11,6 +11,9 @@ class CRUDSkill(CRUDBase[Skill, SkillCreate, SkillUpdate]):
     def get(self, db: Session, skill_id: Any) -> Skill:
         return db.query(self.model).filter(self.model.skill_id == skill_id).first()
 
+    def get_active(self, db: Session, *, skip: int = 0) -> List[Skill]:
+        return db.query(self.model).filter(self.model.is_active == 1).offset(skip).all()
+
```

Comment:
Oops shouldn't be doing a skill specific `get_active` but instead an `is_active` modifier on the crud `get_multi` will be better since that will encompass all crud models

Ref: https://github.com/is212g7t5/spm/pull/44/files

Reply:
done